test/export/test_export.py

```diff
@@ -2476,11 +2464,13 @@ def forward(self, x):
         ) as cm:
             ep.module()(torch.tensor([30]))
 
+    @testing.expectedFailureNonStrict  # assert not found
```

Comment:
Don't we have the deferred runtime assertion now in both non-strict and strict?

Reply:
Not sure.
> RuntimeError: Expected to find "torch.ops.aten._assert_async.msg" but did not find it

Reply:
cc @BoyuanFeng